docs/src/examples/components/Label/Variations/LabelExampleColor.shorthand.tsx

```diff
@@ -0,0 +1,19 @@
+import * as React from 'react'
+import * as _ from 'lodash'
+import { Label, ProviderConsumer } from '@stardust-ui/react'
+
+const LabelExampleColor = () => (
+  <ProviderConsumer
+    render={({ siteVariables: { colorScheme } }) =>
+      _.keys(colorScheme).map(color => (
+        <>
+          <Label key={color} color={color} content={_.startCase(color)} />
+          <br />
+          <br />
```

Comment:
Is there any way to avoid `br`s? May be we can use `Grid` there?

Reply:
introduced Grid, thx